packages/datetime/src/_daterangepicker.scss

```diff
@@ -62,20 +60,26 @@ $daypicker-min-width: $pt-grid-size * 21 !default;
     margin-right: $pt-grid-size;
   }
 
+  // // we only want outside days to be shown when displaying one month at a time
+  // // https://github.com/palantir/blueprint/pull/586/files#r98813760
+  // .DayPicker-Day--outside {
+  //   visibility: hidden;
+  // }
```

Comment:
uncomment? Might want to note that the code doesn't show it already, but the `react-day-picker` still renders the DOM element, which is why we need to hide it.